packages/@aws-cdk/aws-neptune/test/endpoint.test.ts

```diff
@@ -0,0 +1,29 @@
+import { Endpoint } from '../lib';
+
+// A numeric CDK token (see: https://docs.aws.amazon.com/cdk/latest/guide/tokens.html#tokens_number)
+const CDK_NUMERIC_TOKEN = -1.8881545897087626e+289;
```

Comment:
Agh! I see you've copy/pasted this, but let's not repeat this. We can use the `Token` class to deal with tokens. This test otherwise isn't actually testing what you think it does.

Reply:
Removed this test altogether